lib/plugins/aws/package/compile/functions/index.test.js

```diff
@@ -71,6 +71,211 @@ describe('AwsCompileFunctions', () => {
     };
   });
 
+  describe('when function versions are used with layers in the same stack', () => {
```

Comment:
It was an attempt to reference the fact that this is specifically testing the case where layers are defined in the same service, rather than when they already exist and an ARN is used in the function configuration.

However, I realized I can just include a test for the ARN case so I added one and changed the name it to a more general `when function versions are used with layers`.